push.bash

```diff
@@ -37,7 +37,12 @@ if [ -z "$BUF_COMMAND" ]; then
   fail "$NOT_INSTALLED_MESSAGE"
 fi
 
-BUF_ARGS=("--tag" "${GITHUB_SHA}")
+VERSION=${GITHUB_SHA}
+if [ -n "${TAG}" ]; then
+  VERSION="${TAG}"
+fi
```

Comment:
I don't think we should have a default behaviour if no tag is set. It should either be "no tag" or "tag with configured value".